modules/apps/object/object-api/src/main/java/com/liferay/object/exception/RequiredObjectFieldException.java

```diff
@@ -13,9 +13,10 @@
 public class RequiredObjectFieldException extends PortalException {
 
 	public RequiredObjectFieldException() {
-		super("At least one custom field must be added");
+		super("At least one custom or system object field must be added");
```

Comment:
I'm thinking that "At least one object field must be added" could be enough :thinking:

Reply:
What do you think?

Reply:
makes sense